lib/Data/Validate/Sanctions.pm

```diff
@@ -301,6 +296,15 @@ sub _load_data {
     }
     $self->_index_data();
 
+    my %index = $self->{_index}->%*;
+    foreach my $sanctioned_name (keys %index) {
```

Comment:
Is there a special reason to copy this hash ?
```suggestion
    foreach my $sanctioned_name (keys $self->{_index}->%*) {
```

Reply:
Thanks, @chylli-binary.